robot_ws/src/turtlebot_controller/robomaker/inference_worker.py

```diff
@@ -92,8 +99,8 @@ def takeAction(self, action):
     model_path = sys.argv[1]
     print('Starting Inference Worker, Specified Model Directory: ', model_path)
 
-    rospy.init_node('rl_coach', anonymous=True)
+    rclpy.init()
     inference_worker = InferenceWorker(model_path)
     inference_worker.run()
-    rospy.spin()
-
+    rclpy.spin(inference_worker)
+    rclpy.shutdown()
```

Comment:
Before shutdown, call inference_worker.destroy_node()

Reply:
Added in [64bf36f ](https://github.com/aws-robotics/aws-robomaker-sample-application-objecttracker/pull/34/commits/64bf36f8ab18dd2a225c2535c012d396c35644e3) commit.